### PR TITLE
fix: change dropin name to drop-in

### PR DESCRIPTION
**Pull request by bdenham**
This PR changes the spelling of "dropin" to the approved spelling: "drop-in".

Mike Ordoyne from the Brand Strategy team approved the naming convention which should include a hyphen: “drop-in.”

### CODE REVIEW

**Review by jeff-matthews — COMMENTED** [review]
According to the wiki:

- "Drop-in components" is preferred over "DropIns."
the name structure [Functional Name] Drop-in is correct for individual drop-ins. Acronyms should not be used as part of the name. (from Branding)
- "Cart drop-in component", once established, can be written as, "the cart component" or "the cart").

**Comment by bdenham** [comment]
Yep, I’ll do another sweep.

**Comment by bdenham** [comment]
I removed capitalization and created shorthand entries for cart, checkout, and product details drop-ins  as prescribed in the wiki.

File: README.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ The available scripts for running the project are defined in the `package.json`
    ```mdx
    ---
    title: Slots
-   description: Learn about slots and how to use them to customize dropins.
+   description: Learn about slots and how to use them to customize drop-ins.
```
Comment (jeff-matthews) [inline README.md]:
```suggestion
   description: Learn about slots and how to use them to customize drop-in components.
```

File: README.md [hunk 2]
```diff
@@ -89,7 +89,7 @@ The available scripts for running the project are defined in the `package.json`
...
```
Comment (jeff-matthews) [inline README.md]:
```suggestion
   description: Learn about slots and how to use them to customize drop-in components.
```

File: _drafts/_dropins.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
-title: What are dropins?
-description: Learn how dropins define all of our Commerce storefronts.
+title: What are drop-ins?
+description: Learn how drop-ins define all of our Commerce storefronts.
```
Comment (jeff-matthews) [inline _drafts/_dropins.mdx]:
```suggestion
description: Learn how drop-in components define all of our Commerce storefronts.
```

File: _drafts/_dropins.mdx [hunk 4]
```diff
@@ -10,17 +10,17 @@ import { Steps } from '@astrojs/starlight/components';
...
+Drop-in components are full-featured shopping components that turn websites into storefronts. They are not the primitive components, like the Carousels and Galleries used within them. For example, our Product Details, Checkout, and Cart drop-ins are featured as full pages out of the box to create the entire storefront shopping experience for a website. Our drop-ins can also support different layouts for different purposes. For example, our MiniCart is simply a Cart with compact layout typical of MiniCarts so that they can be added to Popover or Panel components.
 
-## Dropins are the building blocks of our storefronts
+## Drop-ins are the building blocks of our storefronts
```
Comment (jeff-matthews) [inline _drafts/_dropins.mdx]:
```suggestion
## Drop-in components are the building blocks of our storefronts
```

File: _dropin-templates/SidebarConfiguration.mjs [hunk 5]
```diff
@@ -1,4 +1,4 @@
-// Use these items to create a TOC entry for a new dropin:
+// Use these items to create a TOC entry for a new drop-in:
```
Comment (jeff-matthews) [inline _dropin-templates/SidebarConfiguration.mjs]:
```suggestion
// Use these items to create a TOC entry for a new drop-in component:
```

File: _dropin-templates/dropin-anatomy.mdx [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 title: DROPIN_NAME anatomy
-description: Learn about the purpose and parts of the DROPIN_NAME dropin.
+description: Learn about the purpose and parts of the DROPIN_NAME drop-in.
```
Comment (jeff-matthews) [inline _dropin-templates/dropin-anatomy.mdx]:
```suggestion
description: Learn about the purpose and parts of the DROPIN_NAME drop-in component.
```

File: _dropin-templates/dropin-anatomy.mdx [hunk 7]
```diff
@@ -12,7 +12,7 @@ import { Steps } from '@astrojs/starlight/components';
 
 ## Anatomy
 
-{/* Replace `placeholder.webp` with a diagram of the dropin identifying all its parts: Containers, slots, components, CSS styles, and design tokens. */}
+{/* Replace `placeholder.webp` with a diagram of the drop-in identifying all its parts: Containers, slots, components, CSS styles, and design tokens. */}
```
Comment (jeff-matthews) [inline _dropin-templates/dropin-anatomy.mdx]:
```suggestion
{/* Replace `placeholder.webp` with a diagram of the drop-in component identifying all its parts: Containers, slots, components, CSS styles, and design tokens. */}
```

File: _dropin-templates/dropin-anatomy.mdx [hunk 8]
```diff
@@ -27,7 +27,7 @@ import { Steps } from '@astrojs/starlight/components';
 
 ## Which parts can I customize?
 
-The DROPIN_NAME dropin provides the following customizable elements:
+The DROPIN_NAME drop-in provides the following customizable elements:
```
Comment (jeff-matthews) [inline _dropin-templates/dropin-anatomy.mdx]:
```suggestion
The DROPIN_NAME drop-in component provides the following customizable elements:
```

**Review by jeff-matthews — COMMENTED** [review]
I started making some suggestions, but there are too may files. I don't think we should use "drop-in" standalone. My understanding is that it should be "drop-in component".

**Comment by bdenham** [comment]
@jeff-matthews I added another pass to changing the standalone "drop-in" instances to "drop-in component"

**Review by jeff-matthews — CHANGES_REQUESTED** [review]
@bdenham, I created #113 to address some of the issues I was still seeing in this PR.

**Review by jeff-matthews — APPROVED** [review]

**Pull request merged**